### DISCUSSION TITLE
Getting started on Precalc.

### DISCUSSION THREAD

**Discussion by deepsa52** [TeamBasedInquiryLearning/library#505] [General]
I am new to this platform. Have used TBIL resources on Precalc. I would like to jump in and help with building precalc activities. What would be a good place to start? I am excited to be here!

**Comment by siwelwerd** [comment]
@AbbyANoble @kathypinzon @tdegeorge I chatted with Deepa for a bit, and suggested she might be able to help review the final trig sections as y'all finish those.  I also suggested she create issues for any suggestions for improvements to the Precalc materials based on her experience using them with her students this fall, and next semester.

If there's any particular places y'all could use help on, drop a note here.

**Comment by AbbyANoble** [comment]
Hi Deepa! Glad to have you here! Reviewing trig would definitely be helpful, as well as giving any feedback on using the activities in your classroom.

**Reply by tdegeorge** [reply]
I agree! Kathy and I are not teaching precalc (we're both at the high school), so it would be super helpful to have someone (in addition to Abby) try them in their class/vet through the activities. Thanks for volunteering!

**Reply by deepsa52** [reply]
Thank you both. I will certainly give feedback on the activities as I use them in the spring, as well as spend sometime next week looking through the activities and think back on how it went when I used them in class this fall. 
Please let me know if I have this correctly:

1) I will create an issue or discussion to communicate my feedback on activities 
2) Create an issue/discussion if I have a suggestion for anything new that could be included
3) This will be an ongoing effort 

As far as reviewing trig material is concerned, how would you like me to get started? I am very eager and I am very happy and excited to be included. 

Also, would it be a good idea to contribute reflections on activities throughout the semester first and then contribute to suggestions for CheckIt exercises question bank afterwards?
